app/scripts/views/sign_up.js

```diff
@@ -177,6 +177,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this.getUserAgent().hasWebChannelSupport()) {
```

Comment:
We also need a unit test change for this test. To run the unit tests go to 127.0.0.1:3030/tests/index.html